### DISCUSSION TITLE
AntennaArray question, Can someone answer this question for me?

### DISCUSSION THREAD

**Discussion by sunskyday** [NVlabs/sionna#630] [Q&A]
Hi,
scene.tx_array = AntennaArray(antenna=Antenna("dipole", "V"),
positions=tf.Variable([[-0.15,0,0], [0.15,0,0]]))

There are two TX antennas, and the two antennas can only work in one pattern and polarization mode. How to define the pattern and polarization of each antenna? Also, the RX antennas has this problem.

**Comment by SebastianCa** [comment]
This is a duplicate of #631
